src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractCreatePdpFeedServiceImpl.java

```diff
@@ -8,17 +8,20 @@
 import edu.cornell.kfs.concur.batch.service.ConcurBatchUtilityService;
 import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractCreatePdpFeedService;
 import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractFileService;
+import edu.cornell.kfs.concur.batch.service.ConcurRequestExtractReportService;
 
 public class ConcurRequestExtractCreatePdpFeedServiceImpl implements ConcurRequestExtractCreatePdpFeedService {
     private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(ConcurRequestExtractCreatePdpFeedServiceImpl.class);
     protected ConcurBatchUtilityService concurBatchUtilityService;
     protected ConcurRequestExtractFileService concurRequestExtractFileService;
+    protected ConcurRequestExtractReportService concurRequestExtractReportService;
 
     @Override
     public void createPdpFeedsFromRequestExtracts() {
         List<String> filesToProcess = getConcurRequestExtractFileService().getUnprocessedRequestExtractFiles();
         if (filesToProcess.isEmpty()) {
             LOG.info("createPdpFeedsFromRequestExtracts: No Request Extract files found to process.");
+            getConcurRequestExtractReportService().sendResultsEmail(null, null);
```

Comment:
Would it be better to add a separate service method for sending a no-files-processed email, so that you don't need to rely on null values and null-checks to do so?